helix-view/src/theme.rs

```diff
@@ -263,19 +264,38 @@ impl ThemePalette {
             .ok_or(format!("Theme: invalid modifier: {}", value))
     }
 
+    pub fn parse_underline_style(value: &Value) -> Result<UnderlineStyle, String> {
+        value
+            .as_str()
+            .and_then(|s| s.parse().ok())
+            .ok_or(format!("Theme: invalid underline_style: {}", value))
+    }
+
     pub fn parse_style(&self, style: &mut Style, value: Value) -> Result<(), String> {
         if let Value::Table(entries) = value {
             for (name, value) in entries {
                 match name.as_str() {
                     "fg" => *style = style.fg(self.parse_color(value)?),
                     "bg" => *style = style.bg(self.parse_color(value)?),
+                    "underline-color" => *style = style.underline_color(self.parse_color(value)?),
+                    "underline-style" => {
+                        warn!("found style");
```

Comment:
Was this meant for debugging? I think it can be removed

Reply:
yeah that was just for debugging and I forgot to remove it, sorry about that!